samples/key_value_api/README.md

```diff
@@ -0,0 +1,30 @@
+# Key-Value Storage API Demonstration
+
+A pipeline demonstrating how to work with the embedded Key-Value store.
+
+Tested on platforms:
+
+- Nvidia Turing, Ampere
+- Nvidia Jetson Orin family
```

Comment:
maybe better to remove orin from tested, below "The demo implemented only for x86.."